### ISSUE TITLE
Test sorting on new UI

### ISSUE THREAD

**Issue by MoeNick** [Giveth/giveth-next#424]
@RamRamez I am trying to test sort by various sections  but all it seems Al of the project have the same amount raised (200 dollars), no pagination only load more option, hearts seems working (UI is broken as we are migration to typescript and it needs to be implemented), all of them created 5 days ago so I cannot test the creation date sorting. And for sorting based on verification it behaves a bit weird: We have trace projects sorted as verified and a project in the middle of the pile of projects as verified:
![image](https://user-images.githubusercontent.com/53400757/142899421-eb0ced6e-3ff8-42cd-920a-4e4ae3be8914.png)
Should I test it later? it seems not ready for testing

_Originally posted by @DonnaOftadeh in https://github.com/Giveth/giveth-next/issues/319#issuecomment-975707231_

**Comment by RamRamez** [comment]
Total amount raised is now implemented in backend and I added to the UI in PR #17 and now it shows the correct value.

No pagination and now we have load more in new design. You can test this instead.

Sorting and search is buggy from backend and @CarlosQ96 is working on it.

Last updated is not implemented yet in backend and @CarlosQ96 is working on it.

You can't test sorting by creation date since it's not shown in project card.

**Comment by CarlosQ96** [comment]
@RamRamez  Working on it. This are issues related to the typeorm library. Will update you soon. 
Also adding the last updated at attribute.

**Comment by MoeNick** [comment]
FYI @CarlosQ96 I push it back to in-progress

**Comment by MoeNick** [comment]
@CarlosQ96 Shall we close it?

**Comment by CarlosQ96** [comment]
@MoeNick Not yet, I am focusing on priorities but this is only for one last bug.
The skip pagination sometimes bugs out, returning values from previous pages. So I talked to Mohammed about a posible solution, introducing another library.
Unless you want to, we can create another separate issue.
Which would be Fix pagination bug.

**Comment by CarlosQ96** [comment]
But everything else Ramin commented on has been fixed.

**Comment by MoeNick** [comment]
@CarlosQ96 We do need good sorting and filtering on launch day and it's the minimum expectation that me and a user can have from an application. let's finish it up. 
I cant find any priorities higher than a bug on production.

**Comment by CarlosQ96** [comment]
I meant, been focusing on the giving blocks, but will tackle this issue next!

**Comment by MoeNick** [comment]
Appreciate it . 
But pls let's not start new issues if we have bugs on production. I really missed this one.

**Comment by mateodaza** [comment]
@MoeNick can we migrate this to the typescript repo?

**Comment by MoeNick** [comment]
Yeah. But Let's make sure sorting works fine on Lauren's version in nextjs. I tested it because we removed pagination, seems it's working.

**Comment by divine-comedian** [comment]
testing on `next.giveth.io` 

I tried to sort by creation date - DESCENDING  
I'm thrown an error and the page crashes 

![image](https://user-images.githubusercontent.com/67759413/145811961-76f0c6c5-8117-4395-b90c-f4741ebcdc94.png)

**Comment by MoeNick** [comment]
@RamRamez are you working in this issue or it's backend?

**Comment by RamRamez** [comment]
> @RamRamez are you working in this issue or it's backend?

Resolved in PR #476:
1. Fix richTextViewer error for null content
2. Fix NextJS config for image sources for new givingBlock images

@divine-comedian Please check it

**Comment by divine-comedian** [comment]
its working well now! 

![image](https://user-images.githubusercontent.com/67759413/145859584-d0d84668-ba8e-49e6-b7f4-a554ce64ee88.png)
